### ISSUE TITLE
localized measurements units

### ISSUE THREAD

**Issue by assafb81** [i18next/i18next#1023]
Hi,

Is there a way to localized measurement units?
show amount in 'Liter' or 'Oz' based on device locale?

Thanks

**Comment by jamuhl** [comment]
eventual numeral has support for that: http://numeraljs.com/

you can use format to include that: https://www.i18next.com/formatting.html

**Comment by assafb81** [comment]
Thanks @jamuhl but I am not looking for numeral value but units value display.
for example: show `250ml` or `8Oz`
if device locale is 'en-us' than the it will show: `8Oz`
if device is 'fr' than it will show: `250ml`

**Comment by jamuhl** [comment]
https://github.com/ben-ng/convert-units might be what you're looking for

**Comment by jamuhl** [comment]
what i mean with it i18next has not built in converters for formats (numbers, quantities,..)

**Comment by assafb81** [comment]
@jamuhl thanks for you quick replay,
Thanks for the references, I will continue looking because that package (https://github.com/ben-ng/convert-units) does the conversions but it doesn't do the based on device locale which is what I really need.

**Comment by jamuhl** [comment]
the problem i see with it...unit conversion is not based on locale/language but on country/region

**Comment by assafb81** [comment]
Ohh,
I didn't know that...I will have to rethink 'Liter' to 'fl-oz' conversion

**Comment by silentwraith6** [comment]
Any updates?
Convert units based on user locale.

**Comment by jamuhl** [comment]
@silentwraith6 nope...but shouldn't be too hard to map a few locales to needed country codes. But sure out of scope for this lib - let's keep things modular
